content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md

```diff
@@ -75,6 +75,11 @@ only the termination message:
 
         kubectl get pod termination-demo -o go-template="{{range .status.containerStatuses}}{{.lastState.terminated.message}}{{end}}"
 
+1. If you are running a multi-container pod, you can use a Go template to include the container's name. By doing so, you can discover which of the containers is failing (please note that the `multi-container-pod` creation is not covered in this exercise):
+
+        kubectl get pod multi-container-pod -o go-template='{{range .status.containerStatuses}}{{printf "%s:\n%s\n\n" .name .lastState.terminated.message}}{{end}}'
```

Comment:
I'd suggest we avoid adding a new item to the list. This information can warrant a standalone paragraph. I don't think we need the notes in the parenthesis. Put it together, I'd write this (with long lines wrapped):

````
If you are running a multi-container pod, you can use a Go template to include the container's name.
By doing so, you can discover which of the containers is failing:

```shell
kubectl get pod multi-container-pod -o go-template='{{range .status.containerStatuses}}{{printf "%s:\n%s\n\n" .name .lastState.terminated.message}}{{end}}'
```

````

Reply:
Makes sense